src/inverted_index/inverted_index.c

```diff
@@ -1216,3 +1216,104 @@ size_t IndexBlock_Repair(IndexBlock *blk, DocTable *dt, IndexFlags flags, IndexR
   IndexResult_Free(res);
   return frags;
 }
+
+void InvertedIndex_RepairBlocks(
+    const InvertedIndex *idx, DocTable *dt,
+    IndexRepairParams *index_params,
+    IndexRepairBlocksParams *block_params
+) {
```

Comment:
Will we have access to `DocTable`, `IndexRepairParams`, and `IndexRepairBlocksParams` in Rust? How much of them are possible to remove?